404.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>404 error page</title>
+    <noscript>
+      <meta http-equiv="refresh" content="0;URL='https://www.adobe.com/404.html'" />
+    </noscript>
+    <script>
+        window.location.href = 'https://www.adobe.com/404.html';
```

Comment:
I would propose `window.location.replace()` in order to skip a browser history entry.

Reply:
good point, will update. thanks, @rofe!